### PR TITLE
Convert SortedDict to OrderedDict (bug 1087029)

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: requirements/prod.txt [hunk 1]
```diff
@@ -76,8 +76,6 @@ ndg-httpsclient==0.3.3
 newrelic==2.20.1.18
 nose==1.3.4
 oauthlib==0.7.2
-# ordereddict is required by kombu
```
Comment (diox) [inline requirements/prod.txt]: Is this no longer true then ?

Comment (robhudson) [inline requirements/prod.txt]: No. I couldn't find the exact version but kombu now requires python 2.7.

Comment (robhudson) [inline requirements/prod.txt]: Here's one example:
https://github.com/celery/kombu/commit/d3a36c6750fafb6806fd027f429fceca556b0263

**Comment by diox** [comment]
r+

**Pull request merged**